### ISSUE TITLE
Connection within Docker

### ISSUE THREAD

**Issue by jmorris644** [gablau/node-red-contrib-blynk-ws#26]
When setting up the connection within Docker I need to use the container name instead of a URL or ip address. When I use the conatiner name and I deploy node-red it throws a warning that the blynk node is not properly configured. Continuing deployment ignores the error and the connection works.

It would be great if the error is not thrown at all.

**Comment by 877dev** [comment]
![image](https://user-images.githubusercontent.com/46575282/94571506-fb8ad980-0267-11eb-8cb8-cc08acecf672.png)

**Comment by jmorris644** [comment]
The internal name actually works and it connects. But node-red throws a bypassable config error when deploying.

**Comment by jmorris644** [comment]
As a temporary fix I commented out the validation code in blynk-ws-client.html for the path field.

**Comment by gablau** [comment]
Hi Joe, 
could you provide me the exact string which is not validated?
These are some valid strings:
```
ws://www.example.it/aaaat
ws://www.example.it
ws://example/aaaat
ws://example
wss://example.it:3000/aaaat
wss://example:3000/aaaat
wss://example:3000
ws://192.168.1.1:3000/aaaaa
ws://192.168.1.1/aaaaa
ws://192.168.1.1
```
best regards

**Comment by jmorris644** [comment]
ws://blynk_server:8080/websockets

**Comment by gablau** [comment]
ok,
the problem is the underscore character that doesn't validate the url.
i'll fix this in next release

**Comment by jmorris644** [comment]
:)

**Comment by gablau** [comment]
fixed in [1.0.4 version](https://github.com/gablau/node-red-contrib-blynk-ws/releases/tag/1.0.4)
